base/applications/osk/lang/pt-PT.rc

```diff
@@ -0,0 +1,84 @@
+/*
+ * PROJECT:         ReactOS Kernel
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * FILE:            base/applications/osk/lang/en-US.rc
+ * PURPOSE:         On screen keyboard (Portuguese resource).
+ * PROGRAMMERS:     Copyright Denis ROBERT
+ * TRANSLATION:     Valmor Flores (valmorflores@gmail.com)
+ */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_DEFAULT
+
+IDR_OSK_MENU MENU DISCARDABLE
+BEGIN
+    POPUP "Ficheiro"
+    BEGIN
+        MENUITEM "&Sair", IDM_EXIT
+    END
+
+    POPUP "Teclado"
+    BEGIN
+        MENUITEM "Teclado extendido", IDM_ENHANCED_KB, CHECKED
+        MENUITEM "Teclado simples", IDM_STANDARD_KB
+        MENUITEM SEPARATOR
+        MENUITEM "Layout padrao", IDM_REG_LAYOUT, CHECKED, GRAYED
+        MENUITEM "Bloquear layout", IDM_BLOCK_LAYOUT, GRAYED
+        MENUITEM SEPARATOR
+        MENUITEM "101 teclas", IDM_101_KEYS, CHECKED, GRAYED
+        MENUITEM "102 teclas", IDM_102_KEYS, GRAYED
+        MENUITEM "106 teclas", IDM_106_KEYS, GRAYED
+    END
+
+    POPUP "Configurações"
+    BEGIN
+        MENUITEM "Sempre no topo", IDM_ON_TOP, CHECKED
+        MENUITEM SEPARATOR
+        MENUITEM "&Ativar som de clique", IDM_CLICK_SOUND
+        MENUITEM SEPARATOR
+        MENUITEM "&Modo digitação...", IDM_TYPE_MODE, GRAYED
+        MENUITEM SEPARATOR
+        MENUITEM "&Fonte...", IDM_FONT
+    END
+
+    POPUP "Ajuda"
+    BEGIN
+        MENUITEM "&Tópicos de ajuda", IDM_HELP_TOPICS, GRAYED
+        MENUITEM SEPARATOR
+        MENUITEM "&Sobre Teclado Virtual...", IDM_ABOUT
+    END
+END
+
+IDD_WARNINGDIALOG_OSK DIALOGEX 0, 0, 250, 97
+STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_POPUP | WS_CAPTION
+CAPTION "Teclado Virtual"
+FONT 8, "MS Shell Dlg", 400, 0, 0x1
+BEGIN
+    DEFPUSHBUTTON "OK", IDOK, 193, 76, 50, 14
+    ICON IDI_OSK, IDC_STATIC, 7, 17, 20, 20
+    LTEXT "Teclado Virtual fornece um nível de funcionalidade para usuários com necessidades especiais. Usuários com dificuldades motoras necessitam de um software para uso diário.", IDC_STATIC, 36, 7, 207, 33
+    CONTROL "Não exibir esta mensagem novamente", IDC_SHOWWARNINGCHECK, "Button",
+            BS_AUTOCHECKBOX | WS_TABSTOP, 43, 80, 137, 10
+END
+
+STRINGTABLE
+BEGIN
+    IDS_OSK "Teclado Virtual"
+    IDS_AUTHORS "Copyright Denis Robert, Valmor Flores"
+    IDS_NUMLOCK "Num"
+    IDS_CAPSLOCK "Caps"
+    IDS_SCROLLLOCK "Scroll"
+    /* Make sure the text fits onto the buttons */
+    IDS_ESCAPE "Esc"
+    IDS_PRN "Prn"
+    IDS_STOP "Stop"
+    IDS_ATTN "Attn"
+    IDS_INSERT "ins"
+    IDS_NUMLOCKKEY "nlk"
+    IDS_DELETE "del"
+    IDS_END "end"
+    IDS_CTRL "ctrl"
+    IDS_LEFTALT "alt"
+    IDS_RIGHTALT "alt"
+END
+
+/* EOF */
```

Comment:
The last line must end with '\n'. So it will be an empty line.
The same for pt-BR file.